Upgrading from 1.8.0 to 1.9.0 and now failing with Message: Not Found

### Describe the bug

I have a learning tutorial (https://kubebyexample.com/learning-paths/developing-spring-boot-kubernetes/lesson-4-deploying-spring-boot-kubernetes-eclipse) which currently uses JKube 1.8.0. It is a simple Spring Boot application that uses the JKube `kubernetes-maven-plugin` to deploy to minikube.

The source code is at https://github.com/RedHat-Middleware-Workshops/spring-jkube-external-config/tree/solution.

As part of the repo there are some GitHub actions which verify that things work correctly. Upon upgrading JKube from 1.8.0 to 1.9.0 the `k8s:deploy` task no longer works. I get this output:

```
╰─ ./mvnw k8s:deploy -Pk8s -DskipTests
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------------< org.acme:spring-jkube-external-config >----------------
[INFO] Building spring-jkube-external-config 0.0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] >>> kubernetes-maven-plugin:1.9.0:deploy (default-cli) > install @ spring-jkube-external-config >>>
[INFO] 
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ spring-jkube-external-config ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 1 resource
[INFO] Copying 0 resource
[INFO] 
[INFO] --- kubernetes-maven-plugin:1.9.0:resource (jkube) @ spring-jkube-external-config ---
[INFO] k8s: Running generator spring-boot
[INFO] k8s: spring-boot: Using Docker image registry.access.redhat.com/ubi8/openjdk-17:latest as base / builder
[INFO] k8s: Using resource templates from /Users/edeandre/workspaces/workshops/spring-jkube-external-config/src/main/jkube
[INFO] k8s: jkube-controller: Adding a default Deployment
[INFO] k8s: jkube-namespace: Adding a default Namespace: spring-jkube-external-config
[INFO] k8s: jkube-healthcheck-spring-boot: Adding readiness probe on port 8080, path='/actuator/health', scheme='HTTP', with initial delay 10 seconds
[INFO] k8s: jkube-healthcheck-spring-boot: Adding liveness probe on port 8080, path='/actuator/health', scheme='HTTP', with initial delay 180 seconds
[INFO] k8s: jkube-service-discovery: Using first mentioned service port '8080' 
[INFO] k8s: jkube-revision-history: Adding revision history limit to 2
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-deployment.yml resource
[WARNING] Unknown keyword $module - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
[WARNING] Unknown keyword existingJavaType - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
[WARNING] Unknown keyword javaOmitEmpty - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-service.yml resource
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-rolebinding.yml resource
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-namespace.yml resource
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-configmap.yml resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.10.1:compile (default-compile) @ spring-jkube-external-config ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ spring-jkube-external-config ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /Users/edeandre/workspaces/workshops/spring-jkube-external-config/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.10.1:testCompile (default-testCompile) @ spring-jkube-external-config ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ spring-jkube-external-config ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-jar-plugin:3.2.2:jar (default-jar) @ spring-jkube-external-config ---
[INFO] Building jar: /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/spring-jkube-external-config-0.0.1-SNAPSHOT.jar
[INFO] 
[INFO] --- spring-boot-maven-plugin:2.7.3:repackage (repackage) @ spring-jkube-external-config ---
[INFO] Replacing main artifact with repackaged archive
[INFO] 
[INFO] --- kubernetes-maven-plugin:1.9.0:build (jkube) @ spring-jkube-external-config ---
[INFO] k8s: Building Docker image in Kubernetes mode
[INFO] k8s: Running generator spring-boot
[INFO] k8s: spring-boot: Using Docker image registry.access.redhat.com/ubi8/openjdk-17:latest as base / builder
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Created docker-build.tar in 137 milliseconds
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Built image sha256:6f7e1
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Removed old image sha256:7f96b
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Tag with latest
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ spring-jkube-external-config ---
[INFO] Installing /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/spring-jkube-external-config-0.0.1-SNAPSHOT.jar to /Users/edeandre/.m2/repository/org/acme/spring-jkube-external-config/0.0.1-SNAPSHOT/spring-jkube-external-config-0.0.1-SNAPSHOT.jar
[INFO] Installing /Users/edeandre/workspaces/workshops/spring-jkube-external-config/pom.xml to /Users/edeandre/.m2/repository/org/acme/spring-jkube-external-config/0.0.1-SNAPSHOT/spring-jkube-external-config-0.0.1-SNAPSHOT.pom
[INFO] Installing /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes.yml to /Users/edeandre/.m2/repository/org/acme/spring-jkube-external-config/0.0.1-SNAPSHOT/spring-jkube-external-config-0.0.1-SNAPSHOT-kubernetes.yml
[INFO] 
[INFO] <<< kubernetes-maven-plugin:1.9.0:deploy (default-cli) < install @ spring-jkube-external-config <<<
[INFO] 
[INFO] 
[INFO] --- kubernetes-maven-plugin:1.9.0:deploy (default-cli) @ spring-jkube-external-config ---
[INFO] k8s: Using Kubernetes at https://127.0.0.1:59256/ in namespace spring-jkube-external-config with manifest /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes.yml 
[INFO] k8s: Creating currentNamespace: spring-jkube-external-config
[INFO] k8s: Created Namespace: target/jkube/applyJson/spring-jkube-external-config/namespace-spring-jkube-external-config-4.json
[INFO] k8s: Creating a Service from kubernetes.yml namespace spring-jkube-external-config name spring-jkube-external-config
[INFO] k8s: Created Service: target/jkube/applyJson/spring-jkube-external-config/service-spring-jkube-external-config-4.json
[INFO] k8s: Creating project: spring-jkube-external-config
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.200 s
[INFO] Finished at: 2022-09-13T08:22:23-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.jkube:kubernetes-maven-plugin:1.9.0:deploy (default-cli) on project spring-jkube-external-config: Execution default-cli of goal org.eclipse.jkube:kubernetes-maven-plugin:1.9.0:deploy failed: Failure executing: GET at: https://127.0.0.1:59256/apis/project.openshift.io/v1/projects. Message: Not Found. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

It looks as if its trying to create a `project.openshift.io/v1/project`, but I'm not sure why it would be doing that. This is the `kubernetes-maven-plugin`, not the `openshift-maven-plugin`.

This issue is preventing me from upgrading.

### Eclipse JKube version

other (please specify in additional context)

### Component

Kubernetes Maven Plugin

### Apache Maven version

3.8.4

### Gradle version

_No response_

### Steps to reproduce

1. Clone https://github.com/RedHat-Middleware-Workshops/spring-jkube-external-config/tree/solution
2. In `pom.xml`, update `<jkube.version>1.8.0</jkube.version>` to `<jkube.version>1.9.0</jkube.version>`
3. Startup minikube
4. Run `./mvnw clean k8s:deploy -Pk8s -DskipTests`
5. Observe error described in description

In addition to failing on my local machine (Mac M1), it is failing in GitHub actions too (https://github.com/RedHat-Middleware-Workshops/spring-jkube-external-config/actions/runs/3034325286)

### Expected behavior

I would expect the deployment to minikube to be successful, just as it is with version 1.8.0

### Runtime

minikube

### Kubernetes API Server version

other (please specify in additional context)

### Environment

Linux, macOS

### Eclipse JKube Logs

```shell
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------------< org.acme:spring-jkube-external-config >----------------
[INFO] Building spring-jkube-external-config 0.0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] >>> kubernetes-maven-plugin:1.9.0:deploy (default-cli) > install @ spring-jkube-external-config >>>
[INFO] 
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ spring-jkube-external-config ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 1 resource
[INFO] Copying 0 resource
[INFO] 
[INFO] --- kubernetes-maven-plugin:1.9.0:resource (jkube) @ spring-jkube-external-config ---
[INFO] k8s: Running generator spring-boot
[INFO] k8s: spring-boot: Using Docker image registry.access.redhat.com/ubi8/openjdk-17:latest as base / builder
[INFO] k8s: Using resource templates from /Users/edeandre/workspaces/workshops/spring-jkube-external-config/src/main/jkube
[INFO] k8s: jkube-controller: Adding a default Deployment
[INFO] k8s: jkube-namespace: Adding a default Namespace: spring-jkube-external-config
[INFO] k8s: jkube-healthcheck-spring-boot: Adding readiness probe on port 8080, path='/actuator/health', scheme='HTTP', with initial delay 10 seconds
[INFO] k8s: jkube-healthcheck-spring-boot: Adding liveness probe on port 8080, path='/actuator/health', scheme='HTTP', with initial delay 180 seconds
[INFO] k8s: jkube-service-discovery: Using first mentioned service port '8080' 
[INFO] k8s: jkube-revision-history: Adding revision history limit to 2
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-deployment.yml resource
[WARNING] Unknown keyword $module - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
[WARNING] Unknown keyword existingJavaType - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
[WARNING] Unknown keyword javaOmitEmpty - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-service.yml resource
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-rolebinding.yml resource
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-namespace.yml resource
[INFO] k8s: validating /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes/spring-jkube-external-config-configmap.yml resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.10.1:compile (default-compile) @ spring-jkube-external-config ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ spring-jkube-external-config ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] skip non existing resourceDirectory /Users/edeandre/workspaces/workshops/spring-jkube-external-config/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.10.1:testCompile (default-testCompile) @ spring-jkube-external-config ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ spring-jkube-external-config ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-jar-plugin:3.2.2:jar (default-jar) @ spring-jkube-external-config ---
[INFO] Building jar: /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/spring-jkube-external-config-0.0.1-SNAPSHOT.jar
[INFO] 
[INFO] --- spring-boot-maven-plugin:2.7.3:repackage (repackage) @ spring-jkube-external-config ---
[INFO] Replacing main artifact with repackaged archive
[INFO] 
[INFO] --- kubernetes-maven-plugin:1.9.0:build (jkube) @ spring-jkube-external-config ---
[INFO] k8s: Building Docker image in Kubernetes mode
[INFO] k8s: Running generator spring-boot
[INFO] k8s: spring-boot: Using Docker image registry.access.redhat.com/ubi8/openjdk-17:latest as base / builder
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Created docker-build.tar in 137 milliseconds
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Built image sha256:6f7e1
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Removed old image sha256:7f96b
[INFO] k8s: [acme/spring-jkube-external-config:latest] "spring-boot": Tag with latest
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ spring-jkube-external-config ---
[INFO] Installing /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/spring-jkube-external-config-0.0.1-SNAPSHOT.jar to /Users/edeandre/.m2/repository/org/acme/spring-jkube-external-config/0.0.1-SNAPSHOT/spring-jkube-external-config-0.0.1-SNAPSHOT.jar
[INFO] Installing /Users/edeandre/workspaces/workshops/spring-jkube-external-config/pom.xml to /Users/edeandre/.m2/repository/org/acme/spring-jkube-external-config/0.0.1-SNAPSHOT/spring-jkube-external-config-0.0.1-SNAPSHOT.pom
[INFO] Installing /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes.yml to /Users/edeandre/.m2/repository/org/acme/spring-jkube-external-config/0.0.1-SNAPSHOT/spring-jkube-external-config-0.0.1-SNAPSHOT-kubernetes.yml
[INFO] 
[INFO] <<< kubernetes-maven-plugin:1.9.0:deploy (default-cli) < install @ spring-jkube-external-config <<<
[INFO] 
[INFO] 
[INFO] --- kubernetes-maven-plugin:1.9.0:deploy (default-cli) @ spring-jkube-external-config ---
[INFO] k8s: Using Kubernetes at https://127.0.0.1:59256/ in namespace spring-jkube-external-config with manifest /Users/edeandre/workspaces/workshops/spring-jkube-external-config/target/classes/META-INF/jkube/kubernetes.yml 
[INFO] k8s: Creating currentNamespace: spring-jkube-external-config
[INFO] k8s: Created Namespace: target/jkube/applyJson/spring-jkube-external-config/namespace-spring-jkube-external-config-4.json
[INFO] k8s: Creating a Service from kubernetes.yml namespace spring-jkube-external-config name spring-jkube-external-config
[INFO] k8s: Created Service: target/jkube/applyJson/spring-jkube-external-config/service-spring-jkube-external-config-4.json
[INFO] k8s: Creating project: spring-jkube-external-config
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.200 s
[INFO] Finished at: 2022-09-13T08:22:23-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.jkube:kubernetes-maven-plugin:1.9.0:deploy (default-cli) on project spring-jkube-external-config: Execution default-cli of goal org.eclipse.jkube:kubernetes-maven-plugin:1.9.0:deploy failed: Failure executing: GET at: https://127.0.0.1:59256/apis/project.openshift.io/v1/projects. Message: Not Found. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```


### Sample Reproducer Project

https://github.com/RedHat-Middleware-Workshops/spring-jkube-external-config/tree/solution

### Additional context

JKube version is `1.9.0` (no option for `1.9.0` is available in the `Eclipse JKube version` dropdown.

Minikube/Kubernetes version:
```
╰─ minikube start                
😄  minikube v1.26.1 on Darwin 12.6 (arm64)
✨  Using the docker driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🚜  Pulling base image ...
🔄  Restarting existing docker container for "minikube" ...
🐳  Preparing Kubernetes v1.24.3 on Docker 20.10.17 ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
    ▪ Using image kubernetesui/dashboard:v2.6.0
    ▪ Using image kubernetesui/metrics-scraper:v1.0.8
🌟  Enabled addons: storage-provisioner, default-storageclass, dashboard
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -21,6 +21,7 @@ Usage:
 ./scripts/extract-changelog-for-version.sh 1.3.37 5
 ```
 ### 1.10-SNAPSHOT
+* Fix #1747: Upgrading from 1.8.0 to 1.9.0 and now failing with Message: Not Found
 
 ### 1.9.0 (2022-09-09)
 * Fix #777: `k8s:build` with Dockerfile throws `Connection reset by peer` error on old docker daemons

--- a/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/OpenshiftHelper.java
+++ b/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/OpenshiftHelper.java
@@ -23,12 +23,10 @@ import java.util.Set;
 import io.fabric8.kubernetes.api.model.HasMetadata;
 import io.fabric8.kubernetes.api.model.KubernetesList;
 import io.fabric8.kubernetes.client.KubernetesClient;
-import io.fabric8.kubernetes.client.KubernetesClientException;
 import io.fabric8.kubernetes.client.utils.Serialization;
 import io.fabric8.openshift.api.model.Parameter;
 import io.fabric8.openshift.api.model.Template;
 import io.fabric8.openshift.client.OpenShiftClient;
-import io.fabric8.openshift.client.OpenShiftNotAvailableException;
 import org.apache.commons.lang3.StringUtils;
 
 /**
@@ -44,11 +42,11 @@ public class OpenshiftHelper {
         if (client instanceof OpenShiftClient) {
             return (OpenShiftClient) client;
         }
-        try {
+
+        if (isOpenShift(client)) {
             return client.adapt(OpenShiftClient.class);
-        } catch (KubernetesClientException | OpenShiftNotAvailableException e) {
-            return null;
         }
+        return null;
     }
 
     public static boolean isOpenShift(KubernetesClient client) {

--- a/jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/OpenshiftHelperTest.java
+++ b/jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/OpenshiftHelperTest.java
@@ -17,7 +17,6 @@ import io.fabric8.kubernetes.api.model.KubernetesList;
 import io.fabric8.kubernetes.api.model.Pod;
 import io.fabric8.kubernetes.api.model.PodBuilder;
 import io.fabric8.kubernetes.client.KubernetesClient;
-import io.fabric8.kubernetes.client.KubernetesClientException;
 import io.fabric8.openshift.api.model.Template;
 import io.fabric8.openshift.api.model.TemplateBuilder;
 import io.fabric8.openshift.client.OpenShiftClient;
@@ -30,7 +29,6 @@ import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
@@ -47,7 +45,7 @@ class OpenshiftHelperTest {
     @Test
     void testAsOpenShiftClientWithNoOpenShift() {
         // Given
-        doThrow(new KubernetesClientException("")).when(kc).adapt(OpenShiftClient.class);
+        when(kc.adapt(OpenShiftClient.class)).thenReturn(oc);
         //When
         OpenShiftClient result = OpenshiftHelper.asOpenShiftClient(kc);
         //Then
@@ -57,6 +55,7 @@ class OpenshiftHelperTest {
     @Test
     void testOpenShiftClientWithAdaptableToOpenShift() {
         // Given
+        when(oc.isSupported()).thenReturn(true);
         doReturn(oc).when(kc).adapt(OpenShiftClient.class);
         //When
         OpenShiftClient result = OpenshiftHelper.asOpenShiftClient(kc);

--- a/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftUndeployServiceTest.java
+++ b/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftUndeployServiceTest.java
@@ -73,6 +73,7 @@ public class OpenshiftUndeployServiceTest {
     // @formatter:off
     new Expectations() {{
       jKubeServiceHub.getClient().adapt(OpenShiftClient.class); result = openShiftClient;
+      openShiftClient.isSupported(); result = true;
     }};
     // @formatter:on
   }
@@ -126,7 +127,8 @@ public class OpenshiftUndeployServiceTest {
     withLoadedEntities(entity);
     // @formatter:off
     new Expectations() {{
-      jKubeServiceHub.getClient().adapt(OpenShiftClient.class); result = null;
+      jKubeServiceHub.getClient().adapt(OpenShiftClient.class); result = openShiftClient;
+      openShiftClient.isSupported(); result = false;
     }};
     // @formatter:on
     // When

